src/Config.php

```diff
@@ -909,6 +909,7 @@ protected function getDefaults()
                 'dbname'         => 'bolt',
                 'prefix'         => 'bolt_',
                 'charset'        => 'utf8',
```

Comment:
Isn't `utf8mb4` a MySQL only work around for their own design mistakes? 

We support Sqlite, PostgreSQL and MySQL is all.